multires-lyrics-search/README.md

```diff
@@ -1,109 +1,180 @@
 
-| Command | Description |
-| :--- | :--- |
-| ``python app.py index`` | To index files/data |
-| ``python app.py search`` | To run query on the index |
-| ``python app.py dryrun`` | Sanity check on the topology |
+Run the following command and leave it open and running in your terminal. 
 
-## View in Browser
+```sh
+python app.py search
+```
+Open a new terminal window and run the following command to launch the web interface. 
 
 ```bash
 cd static
 python -m http.server
 ```
+Now you can open `http://0.0.0.0:8000/` in your browser and you should see a web interface. See the next section to understand your results. 
 
-Open `http://0.0.0.0:8000/` in your browser.
+# Using the search interface and understanding your results
+Let's begin by starting with a simple one-sentence query. For example, if you add the sentence 'I am very happy today; you should see a similar result. Each of these boxes you see on the right-hand side is a song in your dataset. Each highlighted sentence is a 'match.'  A match is a similar sentence, determined by how parallel two vectors are in embedding space.  If you don't understand the concept of embedding spaces, we suggest you check out this guide [here](https://towardsdatascience.com/neural-network-embeddings-explained-4d028e6f0526) and then return to this example after. 
 
-## Use Docker image from the jina hub
+<img width="300" alt="Screenshot 2021-04-30 at 11 56 08" src="https://user-images.githubusercontent.com/59612379/116891772-67cf2080-ac2f-11eb-9fc6-faaaa30e70bb.png">
 
-To make it easier for the user, we have built and published the [Docker image](https://hub.docker.com/r/jinahub/app.example.multireslyricssearch) with 10000 indexed songs (more than the toy example, but just a small part of the huge dataset).
-You can retrieve the docker image using:
 
-```bash
-docker pull jinahub/app.example.multireslyricssearch:0.0.2-0.9.20
-```
-So you can pull from its latest tags.
+Similarity can be adjusted using the breakdown slider on the left-hand side. As you move the slider to the right, you will see more matches appear. This is because we are increasing our radius in the vector space for finding similar matches. 
+
+The relevance score you see at the bottom of the song box summarizes all the matches. Each match has a numeric value, determining how close it is to the original input in the vector space. The average of these match values is the relevance score. This means that a song with a lot of matches will be ranked as highly relevant. 
+
+The example get's a little more complex if you use two or three sentences in the query box. In this case, the query Flow breaks the total input down into 'chunks', which are sentences, but you can adjust this when your building Jina yourself. Finding the relevance score also becomes more complex, but the same basic principle applies. A song with many similar matches will be ranked as relevant, and those without matches will not be.  
```

Comment:
I think this paragraph is when you lose a consistent train of thought. I followed really well until here and then got a bit lost

Reply:
Yeah your right, I updated it. What do you think?